cob_hardware_config/robots/cob4-7/urdf/cob4-7.urdf.xacro

```diff
@@ -24,6 +24,7 @@
 
   <!-- sensors -->
   <xacro:include filename="$(find cob_description)/urdf/sensors/realsense.urdf.xacro" />
+  <xacro:include filename="$(find cob_description)/urdf/sensors/realsense_zr300.urdf.xacro" />
```

Comment:
PR in cob_common is https://github.com/ipa320/cob_common/pull/225

at the moment the files are copied. But as soon as we harmonize hw and sim, we need to add the fisheye camera and if possible the IMU which is not available on the r200